gh-badges/lib/make-badge.js

```diff
@@ -164,7 +164,6 @@ function makeBadge({
     logoPosition,
     logoWidth,
     logoPadding,
-    // `color` and `labelColor` are included for the `_shields_test` template.
```

Comment:
Probably this comment should be kept, and updated to reference the json template.